package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@capacitor-community/firebase-analytics",
-  "version": "0.3.0",
+  "version": "0.3.1",
```

Comment:
Please don't bump the version - this happens during release.

Reply:
changed it bacj